tests/e2e/json_test.go

```diff
@@ -101,6 +101,14 @@ var _ = Describe("odojsonoutput", func() {
 			Expect(areEqual).To(BeTrue())
 
 		})
+
+		// odo project delete -o json
+		It("should be able to delete project", func() {
+			runCmdShouldPass("odo project create project-deletion-test")
+			// validating that it ran with exit status 0
+			runCmdShouldPass("odo project delete project-deletion-test -o json")
```

Comment:
can we check json output  if possible ?

Reply:
its checking for the exit code, not returning any text